colossalai/trainer/hooks/_metric_hook.py

```diff
@@ -346,13 +349,18 @@ def update(self, num_samples, time) -> None:
             self.accumulated_num_samples += self.last_step_num_samples
             self.accumulated_used_time += self.last_step_used_time
 
-    def get_last_step_value(self):
+    def get_last_step_value(self) -> str:
         self.last_step_used_time = all_reduce(self.last_step_used_time, ParallelMode.DATA) / \
             gpc.get_world_size(ParallelMode.DATA)
         self.last_step_num_samples = all_reduce(self.last_step_num_samples, ParallelMode.DATA)
-        return (self.last_step_num_samples / (self.last_step_used_time + 1e-12)).item()
-
-    def get_accumulated_value(self):
+        samplePerSec = _format_number(self.last_step_num_samples / (self.last_step_used_time + 1e-12).item())
```

Comment:
Use underscore case instead of camel case?

Reply:
OK, I fixed it.